sdk/eventhub/azure-eventhub/azure/eventhub/aio/_eventprocessor/event_processor.py

```diff
@@ -310,6 +311,8 @@ def _create_tasks_for_claimed_ownership(
         if not self._running:
             self._running = True
             while self._running:
+                random_jitter = self._load_balancing_interval * random.random() * 0.2
```

Comment:
It seems Java gives a fixed base jitter (2 seconds) time and use `n * BASE` (n in [0,1]) as the jitter.
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventProcessorClient.java#L43

What if `self._load_balancing_interval` is some kind of long, like 1000s?

Reply:
A fixed base jitter 2 seconds would be too large if the load balancing interval is small. A propotional value is better I think. Java fixes the interval at 10 so the base jitter is 2, not a proportional value.